File: 백준/Bronze/32209. 다음 달에 봐요/다음 달에 봐요.py
import sys
input=lambda:sys.stdin.readline().strip()
Q = int(input())
rlt,check = 0,True
while Q:
    a, b = map(int, input().split())
    if a == 1:
        rlt += b
    elif a == 2:
        rlt -= b
    if rlt < 0:
        check = False
    Q -= 1
print("See you next month" if check else "Adios")

File: 백준/Bronze/32209. 다음 달에 봐요/다음 달에 봐요.c
#include <stdbool.h>
#include <stdio.h>
int main(void) {
    int q, a, b, rlt = 0;
    bool check = true;
    scanf("%d", &q);
    while (q) {
        scanf("%d %d", &a, &b);
        if (a == 1)
            rlt += b;
        else if (a == 2)
            rlt -= b;
        if (rlt < 0) check = false;
        q -= 1;
    }
    printf("%s", check ? "See you next month" : "Adios");
    return 0;
}